electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx

```diff
@@ -51,9 +55,9 @@ const FragmentLogs = ({ nodeAddress }) => {
       let { status } = fragment;
 
       if (status && status.Rejected) {
-        status = status.Rejected.reason;
+        status = `Rejected: ${status.Rejected.reason}`;
```

Comment:
do we know the max length of reason? does it displays correctly?

Reply:
For now Ana added a copy button to this field (like in all hashes). If the message is too long it won't be fully displayed in the table, but it can be copied if needed.